src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -107,6 +108,14 @@ namespace AppInstaller::CLI
                 throw CommandException(Resource::String::InvalidArgumentValueError, s_ArgumentName_Scope, { "user"_lis, "machine"_lis });
             }
         }
+
+        if (execArgs.Contains(Args::Type::Locale))
+        {
+            if (!Utility::IsWellFormedBcp47Tag(execArgs.GetArg(Args::Type::Locale)))
+            {
+                throw CommandException(Resource::String::InvalidArgumentValueError, s_ArgumentName_Locale, { "bcp47 language tags"_lis });

```

Comment:
>s_ArgumentName_Locale [](http://example.com/codeflow?start=84&length=21)

If the Arg is defined in another file, this value should come from finding the Arg and using it's name.